### ISSUE TITLE
Does not recognize ILIAS Folders which contain a ":"

### ISSUE THREAD

**Issue by el14s7** [FliegendeWurst/KIT-ILIAS-downloader#3]
Everytime I run the program I get the following error:
```
...\Einführung in die Technische Mechanik II: Dynamik: failed to process URL

Caused by:
    The directory name is invalid. (os error 267)
```
Everything else I have on my ILIAS-Desk still downloads, just that one Module doesnt.
This is probably since it does contain a ':' in it's name!

Great tool by the way! It's very helpful in times of virtual courses!

![Untitled](https://user-images.githubusercontent.com/21080573/101735267-f052f600-3ac1-11eb-9bc9-2e471ecde79a.png)

**Comment by FliegendeWurst** [comment]
Fixed in 0.2.12. I will upload the new executables in a few minutes.

**Comment by el14s7** [comment]
It now works. Thank you very much for your effort!
